integration/app/test-demo/src/pages/RosterTestApp.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { LogLevel } from 'amazon-chime-sdk-js';
+import {
+  MeetingProvider,
+  RosterAttendeeType,
+  useAttendeeStatus,
+  useRosterState,
+} from 'amazon-chime-sdk-component-library-react';
+import MeetingControls from '../containers/MeetingControls';
+import MeetingForm from '../containers/MeetingForm';
+import MeetingRoster from '../containers/MeetingRoster';
+
+export const RosterTestApp: React.FC = () => {
+  const config = {
+    logLevel: LogLevel.INFO,
+  };
+
+  return (
+    <div style={{ display: 'flex', flexDirection: 'column', width: '70%', margin: 'auto' }}>
+      <MeetingProvider {...config}>
+        <h3 data-testid="app-name">Roster Test</h3>
+        <MeetingForm />
+        <Meeting />
+      </MeetingProvider>
+    </div>
+  )
+};
+
+const Meeting: React.FC = () => {
+  return (
+    <>
+      <MeetingControls />
+      <div style={{ marginTop: '2rem', height: '40rem', display: 'flex', flexDirection: 'row' }}>
```

Comment:
Even though this is test demo I like you used correct relevant tags and recommended CSS units like `rem` here. Kudos!